tests/units/Package/PHP/Package.php

```diff
@@ -47,9 +47,9 @@ class Package extends atoum
 
     public function beforeTestMethod($method)
     {
-        $this->packageName = $this->realdom->regex('/\w+/');
```

Comment:
Why removing dynamically generated data?

Reply:
Ah sorry I should have explained, I wasn't expecting someone to read the PR actually ^^

Some atoum features (including this one) relies on a Hoa library that doesn't work on recent PHP versions. I was already deep in the rabbit hole, so I ended up "weakening" the test so that at least we can move forward. To me it sounded an acceptable compromise at the moment.

Reply:
Absolutely, the Hoa dep in Atoum worries me a bit as it seems to do not have found a new maintainer. @Hywan what is your take on it?